packages/manager/modules/dbaas-logs/src/logs/detail/account/setup/logs-account-setup.controller.js

```diff
@@ -4,27 +4,32 @@ export default class LogsAccountSetupCtrl {
     $q,
     $state,
     $stateParams,
+    coreURLBuilder,
     CucControllerHelper,
     CucCloudMessage,
     LogsAccountService,
     LogsHomeService,
     LogsDetailService,
+    LogsConstants,
   ) {
     this.$q = $q;
     this.$state = $state;
     this.$stateParams = $stateParams;
     this.serviceName = this.$stateParams.serviceName;
+    this.coreURLBuilder = coreURLBuilder;
     this.CucControllerHelper = CucControllerHelper;
     this.CucCloudMessage = CucCloudMessage;
     this.LogsAccountService = LogsAccountService;
     this.LogsHomeService = LogsHomeService;
     this.LogsDetailService = LogsDetailService;
+    this.LogsConstants = LogsConstants;
     this.passwordRules = this.LogsAccountService.getPasswordRules(false);
 
     this.initLoaders();
   }
 
   initLoaders() {
+    this.goToIam = this.coreURLBuilder.buildURL('iam', '/#');
```

Comment:
`goTo...` is usually used for state nivagation. Here, it's more a URL
```suggestion
    this.iamUrl = this.coreURLBuilder.buildURL('iam', '/#');
```

Reply:
Ok, updated